WordPress/src/main/java/org/wordpress/android/ui/mysite/menu/MenuActivity.kt

```diff
@@ -0,0 +1,320 @@
+    }
+
+
+    @Suppress("ComplexMethod", "LongMethod")
+    private fun handleNavigationAction(action: SiteNavigationAction?) {
+        when (action) {
+            is SiteNavigationAction.OpenActivityLog -> ActivityLauncher.viewActivityLogList(this, action.site)
+            is SiteNavigationAction.OpenBackup -> ActivityLauncher.viewBackupList(this, action.site)
+            is SiteNavigationAction.OpenScan -> ActivityLauncher.viewScan(this, action.site)
+            is SiteNavigationAction.OpenPlan -> ActivityLauncher.viewBlogPlans(this, action.site)
+            is SiteNavigationAction.OpenPosts -> ActivityLauncher.viewCurrentBlogPosts(this, action.site)
+            is SiteNavigationAction.OpenPages -> ActivityLauncher.viewCurrentBlogPages(this, action.site)
+            is SiteNavigationAction.OpenAdmin -> ActivityLauncher.viewBlogAdmin(this, action.site)
+            is SiteNavigationAction.OpenPeople -> ActivityLauncher.viewCurrentBlogPeople(this, action.site)
+            is SiteNavigationAction.OpenSharing -> ActivityLauncher.viewBlogSharing(this, action.site)
+            is SiteNavigationAction.OpenSiteSettings -> ActivityLauncher.viewBlogSettingsForResult(this, action.site)
+            is SiteNavigationAction.OpenThemes -> ActivityLauncher.viewCurrentBlogThemes(this, action.site)
+            is SiteNavigationAction.OpenPlugins -> ActivityLauncher.viewPluginBrowser(this, action.site)
+            is SiteNavigationAction.OpenMedia -> ActivityLauncher.viewCurrentBlogMedia(this, action.site)
+            is SiteNavigationAction.OpenMeScreen -> ActivityLauncher.viewMeActivityForResult(this)
+            is SiteNavigationAction.OpenUnifiedComments -> ActivityLauncher.viewUnifiedComments(this, action.site)
+            is SiteNavigationAction.OpenStats -> ActivityLauncher.viewBlogStats(this, action.site)
+            is SiteNavigationAction.OpenDomains -> ActivityLauncher.viewDomainsDashboardActivity(
+                this,
+                action.site
+            )
+            is SiteNavigationAction.OpenCampaignListingPage -> activityNavigator.navigateToCampaignListingPage(
+                this,
+                action.campaignListingPageSource
+            )
+            else -> {}
+        }
+    }
+
+    @Composable
+    @SuppressLint("UnusedMaterialScaffoldPaddingParameter")
+    fun UnifiedMenuScreen() {
+        val uiState by viewModel.uiState.collectAsState()
+
+        Scaffold(
+            topBar = {
+                MainTopAppBar(
+                    title = stringResource(id = R.string.my_site_section_screen_title),
+                    navigationIcon = NavigationIcons.BackIcon,
+                    onNavigationIconClick = onBackPressedDispatcher::onBackPressed,
+                )
+            },
+            content = {
+                UnifiedMenuContent(uiState)
+            }
+        )
+    }
+
+    @Composable
+    fun UnifiedMenuContent(uiState: MenuViewState) {
```

Comment:
Hey @zwarm 
I think you missed removing the **Unified** keyword from the compose views, so I did it through the commit 22d70df4277086d405dbe27b230c097eb4879141. Hope thats ok